<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LimeReport::PreviewReportWindow</class>
 <widget class="QMainWindow" name="LimeReport::PreviewReportWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preview</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="leftMargin">
     <number>1</number>
    </property>
    <property name="topMargin">
     <number>1</number>
    </property>
    <property name="rightMargin">
     <number>1</number>
    </property>
    <property name="bottomMargin">
     <number>1</number>
    </property>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoomIn"/>
    <addaction name="actionZoomOut"/>
    <addaction name="separator"/>
    <addaction name="actionShow_Toolbar"/>
   </widget>
   <widget class="QMenu" name="menuReport">
    <property name="title">
     <string>Report</string>
    </property>
    <addaction name="actionPrint"/>
    <addaction name="actionPrint_To_PDF"/>
   </widget>
   <addaction name="menuReport"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>MainToolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionPrint"/>
   <addaction name="actionEdit_Mode"/>
   <addaction name="actionSaveToFile"/>
   <addaction name="actionPrint_To_PDF"/>
   <addaction name="separator"/>
   <addaction name="actionZoomIn"/>
   <addaction name="actionZoomOut"/>
   <addaction name="actionOne_to_one"/>
   <addaction name="actionFit_page_width"/>
   <addaction name="actionFit_page"/>
   <addaction name="separator"/>
   <addaction name="actionFirst_Page"/>
   <addaction name="actionPriorPage"/>
   <addaction name="actionNextPage"/>
   <addaction name="actionLast_Page"/>
   <addaction name="separator"/>
   <addaction name="actionClosePreview"/>
   <addaction name="actionShowMessages"/>
  </widget>
  <widget class="QToolBar" name="editModeTools">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>EditModeTools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSelection_Mode"/>
   <addaction name="actionInsertTextItem"/>
   <addaction name="separator"/>
   <addaction name="actionDelete_Item"/>
  </widget>
  <action name="actionPrint">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/print</normaloff>:/report/images/print</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionZoomIn">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/zoomIn</normaloff>:/report/images/zoomIn</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/zoomOut</normaloff>:/report/images/zoomOut</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionPriorPage">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/prior</normaloff>:/report/images/prior</iconset>
   </property>
   <property name="text">
    <string>Prior Page</string>
   </property>
   <property name="toolTip">
    <string>Prior Page</string>
   </property>
  </action>
  <action name="actionNextPage">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/next</normaloff>:/report/images/next</iconset>
   </property>
   <property name="text">
    <string>Next Page</string>
   </property>
   <property name="toolTip">
    <string>Next Page</string>
   </property>
  </action>
  <action name="actionClosePreview">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/closebox</normaloff>:/report/images/closebox</iconset>
   </property>
   <property name="text">
    <string>Close Preview</string>
   </property>
   <property name="toolTip">
    <string>Close Preview</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
  <action name="actionEdit_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/edit</normaloff>:/report/images/edit</iconset>
   </property>
   <property name="text">
    <string>Edit Mode</string>
   </property>
  </action>
  <action name="actionSaveToFile">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/save</normaloff>:/report/images/save</iconset>
   </property>
   <property name="text">
    <string>Save to file</string>
   </property>
   <property name="toolTip">
    <string>Save to file</string>
   </property>
  </action>
  <action name="actionShowMessages">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="databrowser/lrdatabrowser.qrc">
     <normaloff>:/databrowser/images/error_message_2.png</normaloff>:/databrowser/images/error_message_2.png</iconset>
   </property>
   <property name="text">
    <string>Show errors</string>
   </property>
   <property name="toolTip">
    <string>Show errors</string>
   </property>
  </action>
  <action name="actionFirst_Page">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/first</normaloff>:/report/images/first</iconset>
   </property>
   <property name="text">
    <string>First Page</string>
   </property>
   <property name="toolTip">
    <string>First page</string>
   </property>
  </action>
  <action name="actionLast_Page">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/last</normaloff>:/report/images/last</iconset>
   </property>
   <property name="text">
    <string>Last Page</string>
   </property>
   <property name="toolTip">
    <string>Last Page</string>
   </property>
  </action>
  <action name="actionPrint_To_PDF">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/pdf</normaloff>:/report/images/pdf</iconset>
   </property>
   <property name="text">
    <string>Print To PDF</string>
   </property>
  </action>
  <action name="actionFit_page_width">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/FitWidth.png</normaloff>:/report/images/FitWidth.png</iconset>
   </property>
   <property name="text">
    <string>Fit page width</string>
   </property>
  </action>
  <action name="actionFit_page">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/FitPage.png</normaloff>:/report/images/FitPage.png</iconset>
   </property>
   <property name="text">
    <string>Fit page</string>
   </property>
  </action>
  <action name="actionOne_to_one">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/OneToOne.png</normaloff>:/report/images/OneToOne.png</iconset>
   </property>
   <property name="text">
    <string>One to one</string>
   </property>
  </action>
  <action name="actionShow_Toolbar">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/toolbar</normaloff>:/report/images/toolbar</iconset>
   </property>
   <property name="text">
    <string>Show Toolbar</string>
   </property>
   <property name="toolTip">
    <string>Show toolbar</string>
   </property>
  </action>
  <action name="actionInsertTextItem">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="items/items.qrc">
     <normaloff>:/items/TextItem</normaloff>:/items/TextItem</iconset>
   </property>
   <property name="text">
    <string>InsertTextItem</string>
   </property>
   <property name="toolTip">
    <string>Add new TextItem</string>
   </property>
  </action>
  <action name="actionSelection_Mode">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/editMode</normaloff>:/report/images/editMode</iconset>
   </property>
   <property name="text">
    <string>Selection Mode</string>
   </property>
  </action>
  <action name="actionDelete_Item">
   <property name="icon">
    <iconset resource="report.qrc">
     <normaloff>:/report/images/delete</normaloff>:/report/images/delete</iconset>
   </property>
   <property name="text">
    <string>Delete Item</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="report.qrc"/>
  <include location="items/items.qrc"/>
  <include location="databrowser/lrdatabrowser.qrc"/>
  <include location="report.qrc"/>
  <include location="items/items.qrc"/>
  <include location="databrowser/lrdatabrowser.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionPrint</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotPrint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPriorPage</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotPriorPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNextPage</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotNextPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomIn</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotZoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomOut</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotZoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClosePreview</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFirst_Page</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotFirstPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionLast_Page</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotLastPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPrint_To_PDF</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotPrintToPDF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionInsertTextItem</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotInsertNewTextItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSelection_Mode</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotActivateItemSelectionMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDelete_Item</sender>
   <signal>triggered()</signal>
   <receiver>LimeReport::PreviewReportWindow</receiver>
   <slot>slotDeleteSelectedItems()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotNextPage()</slot>
  <slot>slotPriorPage()</slot>
  <slot>slotZoomIn()</slot>
  <slot>slotZoomOut()</slot>
  <slot>slotPrint()</slot>
  <slot>slotFirstPage()</slot>
  <slot>slotLastPage()</slot>
  <slot>slotPrintToPDF()</slot>
  <slot>slotInsertNewTextItem()</slot>
  <slot>slotActivateItemSelectionMode()</slot>
  <slot>slotDeleteSelectedItems()</slot>
 </slots>
</ui>
